fairseq/models/speech_to_text/convtransformer.py

```diff
@@ -162,7 +162,7 @@ def add_args(parser):
     @classmethod
     def build_encoder(cls, args):
         encoder = ConvTransformerEncoder(args)
-        if getattr(args, "load_pretrained_encoder_from", None):
+        if getattr(args, "load_pretrained_encoder_from", None) is not None:
```

Comment:
nit: is not None condition check is not needed I guess?
```suggestion
        if getattr(args, "load_pretrained_encoder_from", None):
```